products/bigquerydatatransfer/api.yaml

```diff
@@ -98,6 +98,46 @@ objects:
           about the format here:
           https://cloud.google.com/appengine/docs/flexible/python/scheduling-jobs-with-cron-yaml#the_schedule_format
           NOTE: the granularity should be at least 8 hours, or less frequent.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'scheduleOptions'
+        description: |
+          Options customizing the data transfer schedule.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'disableAutoScheduling'
+            description: |
+              If true, automatic scheduling of data transfer runs for this 
+              configuration will be disabled. The runs can be started on ad-hoc 
+              basis using transferConfigs.startManualRuns API. When automatic 
+              scheduling is disabled, the TransferConfig.schedule field will 
+              be ignored.
+            at_least_one_of:
+              - schedule_options.0.disable_auto_scheduling
+              - schedule_options.0.start_time
+              - schedule_options.0.end_time
+          - !ruby/object:Api::Type::String
+            name: 'startTime'
+            description: |
+              Specifies time to start scheduling transfer runs. The first run will be 
+              scheduled at or after the start time according to a recurrence pattern 
+              defined in the schedule string. The start time can be changed at any 
+              moment. The time when a data transfer can be trigerred manually is not 
+              limited by this option.
+            at_least_one_of:
+              - schedule_options.0.disable_auto_scheduling
+              - schedule_options.0.start_time
+              - schedule_options.0.end_time
+          - !ruby/object:Api::Type::String
```

Comment:
Maybe we should use `Time` rather than `String` e.g. `!ruby/object:Api::Type::Time`

https://github.com/GoogleCloudPlatform/magic-modules/blob/897f4572f8640a4dd7b3447a2bdc9ad33a61ab7c/products/sql/api.yaml#L143